### PR TITLE
Search box

### PR DESCRIPTION
**Pull request by aapeliv**
Design:

Screen Shot 2021-04-04 at 12 13 26 pm

Before:

Screen Shot 2021-04-04 at 12 26 32 pm

After:

Screen Shot 2021-04-04 at 3 09 05 pm

### CODE REVIEW

File: app/frontend/src/features/search/SearchBox.tsx [hunk 1]
```diff
@@ -35,13 +38,22 @@ export default function SearchBox() {
   return (
     <>
       <form onSubmit={onSubmit} className={classes.root}>
-        <TextField
-          className={classes.box}
-          id="search-query"
-          name="query"
-          label="Search"
-          inputRef={register}
-        ></TextField>
```
Comment (lucaslcode) [inline app/frontend/src/features/search/SearchBox.tsx:44]: just use this with variant="standard"

File: app/frontend/src/features/search/SearchBox.tsx [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import { makeStyles } from "@material-ui/core";
-import TextField from "components/TextField";
+import FormControl from "@material-ui/core/FormControl";
+import IconButton from "@material-ui/core/IconButton";
+import Input from "@material-ui/core/Input";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import InputLabel from "@material-ui/core/InputLabel";
+import SearchIcon from "@material-ui/icons/Search";
```
Comment (lucaslcode) [inline app/frontend/src/features/search/SearchBox.tsx]: prefer importing from `@material-ui/core`

**Review by lucaslcode — CHANGES_REQUESTED** [review]

File: app/frontend/src/features/search/SearchBox.tsx (hunk 1 above)
Comment (aapeliv) [reply app/frontend/src/features/search/SearchBox.tsx:44]: It doesn't allow adding the little magnifying glass, I don't think? That's why I had to do the form control stuff

File: app/frontend/src/features/search/SearchBox.tsx (hunk 2 above)
Comment (aapeliv) [reply app/frontend/src/features/search/SearchBox.tsx]: Did I do it right?

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**